packages/gestalt/src/OverlayPanel.js

```diff
@@ -2,41 +2,41 @@
 
 import { type Node } from 'react';
 import AnimationProvider from './AnimationContext.js';
-import InternalSheet from './InternalSheet.js';
+import InternalOverlayPanel from './InternalOverlayPanel.js';
 
 type NodeOrRenderProp = Node | (({| onDismissStart: () => void |}) => Node);
 
 type Props = {|
   /**
-    Supply a short, descriptive label for screen-readers as a text alternative to the Dismiss button. See the [Accessibility section](https://gestalt.pinterest.systems/web/sheet#Accessibility) for more info.
+    Supply a short, descriptive label for screen-readers as a text alternative to the Dismiss button. See the [Accessibility section](https://gestalt.pinterest.systems/web/overlaypanel#Accessibility) for more info.
    */
   accessibilityDismissButtonLabel?: string,
   /**
-   * Supply a short, descriptive label for screen-readers to contextualize the purpose of Sheet. See the [Accessibility section](https://gestalt.pinterest.systems/web/sheet#Accessibility) for more info.
+   * Supply a short, descriptive label for screen-readers to contextualize the purpose of OverlayPanel. See the [Accessibility section](https://gestalt.pinterest.systems/web/overlaypanel#Accessibility) for more info.
    */
-  accessibilitySheetLabel: string,
+  accessibilityOverlayPanelLabel: string,
```

Comment:
same question about this prop name

Reply:
MMM Ok, this one might make sense, as accessibilityLabel is usually applied to the main component